Jumpbox provisioner changes

JermilaPaulDhas opened the pull request.

sbuliarca commented on content-jumpbox-provisioner/cloudformation/stack.yml:
```diff
@@ -101,22 +117,34 @@
+            chmod +x /usr/local/bin/list-clusters
+            cd /home/ec2-user
+            # Install kubectl
+            wget https://storage.googleapis.com/kubernetes-release/release/v1.7.6/bin/linux/amd64/kubectl
```
I think we should be using a newer version of kubectl, like 1.10

JermilaPaulDhas replied:
Done. Also updated the k8s-cli-utils https://github.com/Financial-Times/k8s-cli-utils/pull/9

sbuliarca commented on content-jumpbox-provisioner/k8s_ssh/ssh_config:
```diff
@@ -0,0 +1,66 @@
+        SendEnv LC_IDENTIFICATION LC_ALL LANGUAGE
+        SendEnv XMODIFIERS
+        
+Host *-tunnel-up.ft.com
```
I don't think we use these host names any more.

sbuliarca commented on content-jumpbox-provisioner/k8s_ssh/k8s_clusters.sh:
```diff
@@ -0,0 +1,13 @@
+list-clusters
+
+To connect to a cluster:
+source cluster-login.sh upp-delivery-prod-eu
```
Probably we also need to instruct them to open the outputted URL in a browser and to copy the token afterwards.

JermilaPaulDhas commented:
@sbuliarca Updated the PR with the following changes:
* Removed the ssh config coz ops are not expected to login to our tunnel boxes to ssh to the worker instances
* Added instructions for Ops to copy past url and token
